uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -23,7 +23,6 @@ <h4 class="modal-title">
         autofocus
         ng-model="$ctrl.request.contact_id"
         ng-change="$ctrl.initAfterContactSelection()"
-        theme="civihr-ui-select"
```

Comment:
why does removing the theme works? If the theme was specified, and if before the dropdown looked correct, doesn't it mean the theme was necessary?